Reduce motor currents

dandancheeseandham opened the pull request:
Reduce motor currents and also reduce Z speed, to reduce damage in a crash.

dandancheeseandham committed:
Reduce motor currents

Reduce motor currents and also reduce Z speed, to reduce damage in a crash.

lostapathy commented:
Do we want to lower the M203 on X/Y as well while we're doing this?

Otherwise I think this is a great change

dandancheeseandham committed:
More conservative

M558 mostly removed , as it's configured in homez.g and speeds are beholden to maximum speeds set in M203.
Just left dive height.
X,Y max speeds, acellerations and jerk all made more conservative :)

dandancheeseandham commented:
M558 mostly removed , as it's configured in homez.g and speeds are beholden to maximum speeds set in M203.
Just left dive height.
X,Y max speeds, accelerations and jerk all made more conservative :)

dandancheeseandham commented:
Oh, and I've tested this, it seems to work nicely, and home with stallguard/sensorless and a highish preload without false positive.
